services/wallet/router/router_send_type.go

```diff
@@ -103,9 +103,15 @@ func (s SendType) needL1Fee() bool {
 	return !s.IsEnsTransfer() && !s.IsStickersTransfer()
 }
 
+// This function is used to check if certain SendType can be used with a given path processor
```

Comment:
Quick one: the convention in Go is to start the comment with the name of the function.